include/vcpkg/base/chrono.h

```diff
@@ -8,9 +8,11 @@
 
 namespace vcpkg::Chrono
 {
-    class ElapsedTime
+    struct ElapsedTime
     {
+    private:
         using duration = std::chrono::high_resolution_clock::time_point::duration;
```

Comment:
This name is used in the ctor so I think it should be public

Reply:
:+1: